src/components/NumberInput/NumberInput.stories.tsx

```diff
@@ -79,7 +79,7 @@ export const WithDecoratorAndIncrementable: StoryFn<NumberInputProps> = (args) =
 export const WithErrorLessThanZero: StoryFn<NumberInputProps> = (args) => {
     const [status, setStatus] = useState<Validation>(Validation.Success);
     const [valueSelect, setValueSelect] = useState<boolean>(false);
-    const handleValidation = (value: number) => {
+    const handleValidation = (value: number = 0) => {
```

Comment:
If you pass undefined, value will turn to 0

Reply:
True, but i set a `default` value to avoid the `type` check issues in the conditions that follow